api-tests/apply_route_test.go

```diff
@@ -403,10 +408,12 @@ func TestApplyPsmdbReturnedVersions(t *testing.T) {
 		{"recommended", "1.7.0", nil, "4.4.3-5"},
 		{"recommended", "1.6.0", nil, "4.4.2-4"},
 		{"recommended", "1.5.0", nil, "4.2.8-8"},
+		{"recommended", "1.16.0", &v50, "6.0.15-12"},
 		{"recommended", "1.15.0", &v50, "5.0.20-17"},
 		{"recommended", "1.14.0", &v50, "5.0.15-13"},
 		{"recommended", "1.13.0", &v50, "5.0.11-10"},
 		{"recommended", "1.12.0", &v50, "5.0.7-6"},
+		{"recommended", "1.16.0", &v44, "5.0.26-22"},
```

Comment:
it cannot be 5.0, it needs to be some 4.4 version, but we don't support 4.4 any more so this assertion needs to be removed